client/src/mockData.js

```diff
@@ -0,0 +1,57 @@
+
+// THIS IS JUST MOCK DATA FOR TESTING AND DEVELOPMENT PURPOSES
+
+var Users = [
+    {
+        userId: 2,
+        name: "Elder Gnome",
+        username: "elderGnome",
+        email: "eldergnome@example.com",
+        password: "HAHAHAHA",
+        age: 99,
+        lastLogin: "01-05-2022",
+        isLoggedIn: false
+    },
+    {
+        userId: 3,
+        name: "Chicken",
+        username: "chicken656",
+        email: "chicken656@example.com",
+        password: "POKPOKPOK",
+        age: 13,
+        lastLogin: "07-06-2022",
+        isLoggedIn: false
+    },
+    {
+        userId: 4,
+        name: "Clarita",
+        username: "zombie999",
+        email: "zomvie999@example.com",
+        password: "BRAINS!",
+        age: 55,
+        lastLogin: "23-04-2022",
+        isLoggedIn: false
+    },
+    {
+        userId: 5,
+        name: "Baby bird",
+        username: "babybird123",
+        email: "babybird@example.com",
+        password: "123456789",
+        age: 25,
+        lastLogin: "11-05-2022",
+        isLoggedIn: false
+    },
+    {
+        userId: 6,
+        name: "LOL CAT",
+        username: "lolcat987",
+        email: "lolcat@example.com",
+        password: "987654321",
+        age: 33,
+        lastLogin: "20-03-2022",
+        isLoggedIn: false
+    }
+];
+
+export default Users;
```

Comment:
This is great!

Reply:
this will be provided by ORM

Reply:
Given the shape of data (POJOs), I suggest to use [MongoDB](https://www.mongodb.com/) and [Mongoose](https://mongoosejs.com/) as ORM. Least time to setup, free, awesome toolkit and minimal [impedance mismatch](https://en.wikipedia.org/wiki/Object%E2%80%93relational_impedance_mismatch) (aka butthurt) guaranteed. 😊